app/assets/stylesheets/govuk_publishing_components/components/_big-number.scss

```
@@ -1,11 +1,12 @@
 .gem-c-big-number {
   margin-bottom: govuk-spacing(3);
-  font-family: $govuk-font-family;
+  @include govuk-typography-common($font-family: $govuk-font-family);
+  color: $govuk-text-colour;
 }
 
 .gem-c-big-number__value {
   font-size: 80px;
-  font-weight: 700;
+  @include govuk-typography-weight-bold($important: false);
```

No need to pass `$important: false` – that's the default behaviour.

```suggestion
  @include govuk-typography-weight-bold;
```